### ISSUE TITLE
Could not load type Meziantou.Analyzer.Rules.NullableAttributeUsageAnalyzer

### ISSUE THREAD

**Issue by davidkallesen** [meziantou/Meziantou.Analyzer#339]
In build/nuget 1.0.701 I have no problem. - But upgrade to 1.0.702 or latest, the I get the following error for each project in my solution:

Warning	CS8032	An instance of analyzer Meziantou.Analyzer.Rules.NullableAttributeUsageAnalyzer cannot be created from C:\Users\David\.nuget\packages\meziantou.analyzer\1.0.702\analyzers\dotnet\cs\Meziantou.Analyzer.dll: Could not load type 'Meziantou.Analyzer.Rules.NullableAttributeUsageAnalyzer' from assembly 'Meziantou.Analyzer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'

**Comment by meziantou** [comment]
This error can happen when you use an older version of Roslyn / Visual Studio.

- Does the project build with `dotnet build`? Are there warnings about analyzers?
- What's the output of `dotnet --info`?
- What's the full version of Visual Studio? (Available in Help / About Microsoft Visual Studio)
- Is there an update of Visual Studio available?
- What is the version of Roslyn? You can add `#error version` in a cs file of your project. This should report the version of Roslyn in the error window or in the build output. See [Getting Roslyn (C# compiler) and Language versions](https://www.meziantou.net/getting-roslyn-csharp-compiler-and-language-versions.htm) for more information

**Comment by davidkallesen** [comment]
I have now updated all nuget packages in my solution (including all analyzers - VS2022, ReSharper and .NET was latest versions), then I made a clean-and-rebuild 2-3 time with the same error. Then I made a restarting of VS2022 and then clean-and-rebuild again 2-3 times with the same error.
Then I gave up, left the computer for 1-2 hours and came back - and now it works without the CS8032 warnings... I cant explain what and why... but now it works and right now I cant reproduce the error.

So let close this issue, and hope it is solved by magic.
